serdaSimus/assignments/slidingwindow/lc53/lc53.js

```diff
@@ -1,7 +1,59 @@
+// Problem Statement #
+// Given an array of positive numbers and a positive number ‘k,’ find the maximum sum of any contiguous subarray of size ‘k’.
+
+/*
+  [] ... [1] ... [1,2,3,4,5]
+  I need accumulator: sum
+  I need var: largestValue
+  I need a counter: 
+
+  for loop: 
+    add each item to sum
+    check if counter >= k:
+      compare sum and largest: if sum is bigger:
+        i would save sum to larges, else I would continue
+        sum = sum - arr[counter - k]
+    counter++
+
+  return largestValue
+      
+*/
```

Comment:
High level overview has been one of the better I have seen! You are very close. Make sure it is more conversational like a short story about the steps you will take. This will help other coding languages understand truly what you are up to! 

Good work! #LETSGO